### PR TITLE
add: more doc about routes and make JWS algo more clear

### LINKED ISSUE #30: document new features
* e.g. how to add custom routes

### PR DESCRIPTION
**Pull request by ybelMekk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +205,25 @@ val wellKnownUrl = server.wellKnownUrl("yourissuer")
 // http://localhost:<a random port>/yourissuer/.well-known/openid-configuration
 ```
 
+##### Routes
+
+You can add and customize http routs for your authorization server.
```
Comment (jksolbakken) [inline README.md:210]: Spelling, routes

**Review by jksolbakken — APPROVED** [review]
Just some nitpicking on language and formatting, looks good otherwise!

File: README.md [hunk 2]
```diff
@@ -319,6 +332,44 @@ and return a token response containing a token with the following claims:
...
 ```
 
+A token provider can support different `signing` algorithms. Configure your token provider and
+add this to your config with preferred `JWS algorithm` with preferrd `first` token callback:
+
+```json
+{
+  "tokenProvider": {
+    "keyProvider": {
+      "initialKeys": "{\"kty\": \"EC\",\"d\": \"o9INzHyU_I97djF36YQRpHCJxFTgDTbS1OtwUnHc34U\",\"use\":\"sig\",\"crv\": \"P-256\",\"kid\": \"issuer0\",\"x\": \"umybCYzE-VX_UAIJaX3wc-GTOgB7WDp7A3JJAKW_hqU\",\"y\": \"m_sCzuMjiBSQ7At9yNktMQvE1cCKq68jO7wnRczwKw8\"}",
```
Comment (jksolbakken) [inline README.md:342]: Remove escaping on key name quotes?

Comment (ybelMekk) [reply README.md:342]: Not sure what you mean, could be, i can see its done like this for the other examples?

**Comment by ybelMekk** [comment]
@tommytroen ? 🤓

**Pull request closed without merging**